rules/aip0136/response_message_name_test.go

```diff
@@ -30,7 +30,8 @@ func TestResponseMessageName(t *testing.T) {
 			problems        testutils.Problems
 		}{
 			{"Valid", "ArchiveBook", "ArchiveBookResponse", testutils.Problems{}},
-			{"Invalid", "ArchiveBook", "ArchiveBookResp", testutils.Problems{{Suggestion: "ArchiveBookResponse"}}},
+			{"Invalid", "ArchiveBook", "ArchiveBookResp", testutils.Problems{{
+				Message: "Custom methods should return a message matching the RPC name, with a `Response` suffix, or the resource being operated on, not \"ArchiveBookResp\"."}}},
```

Comment:
Nit: the `testutils.Problems` `Diff` method actually does a `strings.Contains`, so we can pick a chunk of the expected problem message and just use that e.g. 
```suggestion
				Message: "matching the RPC name, with a `Response` suffix}}},
```

https://github.com/googleapis/api-linter/blob/a7290a746589155df64410cdc8267e1ede1a694e/rules/internal/testutils/problems.go#L54-L58

might seem odd but makes for easier/cleaner tests, especially considering there is only one rule being run here so no chance to confuse with another rule

Reply:
Oh, cool.  Good to know.  I'll sneak that in really quick. Also qq about releases.  I noticed this didn't fire off a release: https://github.com/googleapis/api-linter/commit/46a6e43d1d6bb6a6e79131866f16f0b1dfd2e326

Should I rename this PR before it's merged?  Is it because of the (AIP-136) in the PR title?

Reply:
> Also qq about releases. I noticed this didn't fire off a release: https://github.com/googleapis/api-linter/commit/46a6e43d1d6bb6a6e79131866f16f0b1dfd2e326
>
> Should I rename this PR before it's merged? Is it because of the ([AIP-136](https://google.aip.dev/136)) in the PR title?

Nope, all is WAI, it's in the open release PR: https://github.com/googleapis/api-linter/pull/1388

I just haven't pressed the buttons yet knowing we'd have this.